### ISSUE TITLE
[Docs]: Need thumbnail pictures for the blogs

### ISSUE THREAD

**Issue by yzeng25** [iresty/iresty.github.io#1245]
### Current State

**Description of the current state/issue**:

The default thumbnail picture for each blog is the same. I think this is an issue, since it just looks weird, and each blog should have a unique cover picture.

**Link to the page**:

https://apisix.apache.org/blog/

**Screenshots (if relevant)**:

<img width="1920" alt="image" src="https://user-images.githubusercontent.com/36651058/181478769-4e6a2a62-605f-467c-a96e-b20b6be9eed1.png">

<img width="1920" alt="image" src="https://user-images.githubusercontent.com/36651058/181478849-fe92ff0c-d2a5-4d95-b8b9-392d14d8ab48.png">

### Desired State

**Description of the change you are proposing**:

Each blog should have a unique cover picture.

### Would you like to fix this issue?

No. Please assign another contributor

**Comment by adarrssh** [comment]
Please assign me this issue.
What kind of picture would you like to add to other blogs?

**Comment by yzeng25** [comment]
> Please assign me this issue. What kind of picture would you like to add to other blogs?

My first thought is:

- Each blog post has at least one picture in its content.
- Grab the most suitable picture from the content and make it the cover.
- Or generate a cover picture on your own

Due to the recent refactorization, I am not sure how it works. You would need some consultations from @SkyeYoung .

**Comment by SkyeYoung** [comment]
@adarrssh @yzeng25 

Before we can do that, we need to do a few more things. (This part of the work will be logged to https://github.com/apache/apisix-website/issues/1048#issuecomment-1195504988 when it is completed )

- [x] update the project documentation https://github.com/apache/apisix-website/pull/1263
- [x] test and adjust how the thumbnail pictures should appear in the blog
	- The thumbnails will only be displayed on the list page, not in the blog post.

So, please wait a little bit, probably tomorrow we can start the work on replacing the images.

And about these images, it is very obvious that it should express the main idea of the article. We need @hf400159 @SylviaBABY  @juzhiyuan 's help too.

If it needs to be described more explicitly, it's something like the images in https://blog.postman.com/. But please don't imitate it directly, maybe we can discuss and try to make some images first.

**Comment by yzeng25** [comment]
Yeah indeed. The final result is upload the images, but before that, there are tons of other work to do. @adarrssh please wait for those work are ready. We can consider you call dibs on this issue. When everything is ready, and if you are still interested in solving it, then go ahead and do it.

**Comment by adarrssh** [comment]
Okay, I'll wait no problem 🙂

**Comment by SkyeYoung** [comment]
@adarrssh Hi, most of the preparation work has been done.

🔔  Could you please give us some sketches or full images to give us an idea of what you think of them?

***

Here are some of our current settings for the default image: 

The original size of the image is 768x407.
<img width="378" alt="image" src="https://user-images.githubusercontent.com/48400568/183325592-2b34f3e7-6697-4fd2-b7b0-bfdf14a279a8.png">

It is important to note that the images in [the first page](https://apisix.apache.org/blog/) of the blog list pages should perform as well as possible in the 601x232 style.

<img width="330" alt="image" src="https://user-images.githubusercontent.com/48400568/183324806-c2255cf3-8aea-43ca-b5f7-1a28bb5d7ffd.png">

🌟 I think these settings may not be reasonable, you are welcome to suggest your ideas for images and styles.

**Comment by adarrssh** [comment]
Hey @SkyeYoung,  My Univesity exam has started and I have a pretty busy schedule for the next 2 weeks, if you can complete this issue then you can do it, or we can wait for some time so that I can finish this issue later.

**Comment by SkyeYoung** [comment]
@adarrssh Good luck with your exams. We are working on some pictures to solve part of this issue. You are welcome to join us after the exam to solve this issue!

**Comment by BhavyaT-135** [comment]
Hi @SkyeYoung, could I contribute to this issue if no one else is working on it at the moment? I plan to use Canva to create thumbnail images relevant to the topics and would like to discuss the images for each blog individually so that I can improve them according to feedback😊

**Comment by SkyeYoung** [comment]
@BhavyaT-135 You're more than welcome to contribute to this project! No one else is currently working on this issue, so I think you can be as creative as you want. If you have any questions or need help, please feel free to @ me, or anyone else. 😸

**Comment by BhavyaT-135** [comment]
Hi @SkyeYoung, for starters, I created the thumbnail for [Integrate API Gateway with Eureka](https://apisix.apache.org/blog/2022/03/05/apisix-integration-eureka-service-discovery/) blog. I kept the design minimalistic. Kindly take a look and suggest any changes that you would like to see. 😊 

![apisix-eureka](https://user-images.githubusercontent.com/55613637/188264464-f42fbdfb-3fcc-4a42-8138-28036e5287b5.png)

**Comment by SkyeYoung** [comment]
> Hi @SkyeYoung, for starters, I created the thumbnail for [Integrate API Gateway with Eureka](https://apisix.apache.org/blog/2022/03/05/apisix-integration-eureka-service-discovery/) blog. I kept the design minimalistic. Kindly take a look and suggest any changes that you would like to see. 😊
> 
> ![apisix-eureka](https://user-images.githubusercontent.com/55613637/188264464-f42fbdfb-3fcc-4a42-8138-28036e5287b5.png)

I think it's good enough design. 

@juzhiyuan @hf400159 @yzeng25  Do you have any suggestion?

**Comment by yzeng25** [comment]
Generally looks good to me. Just some thoughts:

1. Could you change the major background color to red/white/grey/blue/purple/orange-ish? The current dark green-ish background just does not match with the website's theme colors.
2. Move the "~~~" upward a bit, so that the APISIX logo and the lines are separated.
3.  Eureka has its own logo, you can see this [search result](https://www.google.com.hk/search?q=Eureka&newwindow=1&sxsrf=ALiCzsZlGwSZgdvj-TonWsApPEIDX2xfXA:1662348605137&source=lnms&tbm=isch&sa=X&ved=2ahUKEwjPleiQ2_z5AhUH7jgGHb9zC9IQ_AUoAXoECAIQAw&biw=1728&bih=967&dpr=2#imgrc=0pZlaqPHw0vWcM&imgdii=ERxfBGb1ta-cQM), so I suggest replace the current spring logo with it.

**Comment by BhavyaT-135** [comment]
Hi @SkyeYoung @yzeng25, thanks for the review! I've made the changes and used an orange gradient background. Kindly take a look and let me know if this looks better! 😊 

![apisix-eureka-2](https://user-images.githubusercontent.com/55613637/188381736-7137aace-0d05-425a-918c-b7edac2b9b7c.png)

**Comment by yzeng25** [comment]
LGTM. Any other thoughts @SkyeYoung @juzhiyuan ?

**Comment by Zerozero-123** [comment]
I think the second picture background color with a similar color so that the logo is not very prominent, compared to the first color makes the content more prominent, green background and Eureka logo has echoes of green, you can change the two pictures on the article to see the preview, it is easier to see which is better. @yzeng25

**Comment by SkyeYoung** [comment]
> I think the second picture background color with a similar color so that the logo is not very prominent, compared to the first color makes the content more prominent, green background and Eureka logo has echoes of green, you can change the two pictures on the article to see the preview, it is easier to see which is better. @yzeng25

@BhavyaT-135

**Comment by BhavyaT-135** [comment]
Hi @SkyeYoung @yzeng25, I tried setting up the project on my local machine but am running into some issues. I have contributed to the project before and it was working fine back then. I don't understand what the problem is now. 😅 

For the time being, I tried the preview of the two images on the official website using Google developer tools. Kindly take a look 😃 

![apisix-orange](https://user-images.githubusercontent.com/55613637/188588023-2e27cc82-6f16-4905-aabb-017c982a06f3.png)

![apisix-green](https://user-images.githubusercontent.com/55613637/188588109-7879385d-2ddb-4338-b463-4b2eb0bf95d1.png)

**Comment by SkyeYoung** [comment]
> I tried setting up the project on my local machine but am running into some issues.

@BhavyaT-135 Could you please upload the screenshots of the errors here so that I can assist you in solving these problems?

**Comment by BhavyaT-135** [comment]
@SkyeYoung After cloning the repo and setting up remotes, I ran the command `yarn` and faced the following error in the end.

![yarn error](https://user-images.githubusercontent.com/55613637/188600968-e5acefcb-4f1f-4481-9c2c-6e27b9a40233.png)

Then, I ran the command `yarn prepare-data` and it executed perfectly. When I run `yarn start:doc` and `yarn start:website`, everything works. But, when I run the command `yarn start:blog:en`, I get this error:

![start-blog-error](https://user-images.githubusercontent.com/55613637/188601676-121e29fe-51e2-4e59-b6aa-b404e2b1cb49.png)

**Comment by BhavyaT-135** [comment]
When I run the command `yarn build:preview`, the following error is shown.

![image](https://user-images.githubusercontent.com/55613637/188607265-f9d0b4be-4146-4fb7-8833-b458500748a2.png)

I just figured that the script for `yarn buld:preview` is `preview=true yarn run build`. However, when I manually type the script `yarn run build preview=true`, my build process starts without error. I'll let you know if everything works fine after running `yarn serve`. 😊

**Comment by SkyeYoung** [comment]
> Then, I ran the command `yarn prepare-data` and it executed perfectly. When I run `yarn start:doc` and `yarn start:website`, everything works. But, when I run the command `yarn start:blog:en`, I get this error:
> 
> ![start-blog-error](https://user-images.githubusercontent.com/55613637/188601676-121e29fe-51e2-4e59-b6aa-b404e2b1cb49.png)

Sorry to be late. I think I know what the cause of these problems is.

It's because I was using the command `cp -r` to copy some files, but this command doesn't exist in windows.

> When I run the command `yarn build:preview`, the following error is shown.
> 
> ![image](https://user-images.githubusercontent.com/55613637/188607265-f9d0b4be-4146-4fb7-8833-b458500748a2.png)
> 
> I just figured that the script for `yarn buld:preview` is `preview=true yarn run build`. However, when I manually type the script `yarn run build preview=true`, my build process starts without error. I'll let you know if everything works fine after running `yarn serve`. 😊

The reason for this problem is similar. In Windows, the syntax of cmd or powershell is not the same as the shell in Linux.

I think this is due to the fact that most of the maintainers of this project are developing with Linux or MacOS, which causes these problems.

As a solution, maybe you can try the new development method I added https://github.com/apache/apisix-website#getting-started-in-a-dev-container so that you can use a Linux-like development experience.

Or, please wait a day or two, these two issues are not that hard to fix and I can submit a PR to fix the problems. (But I'm not sure if u'll encounter any other problems 😅)

**Comment by BhavyaT-135** [comment]
Sure @SkyeYoung, I'll wait! 😊

**Comment by SkyeYoung** [comment]
> Sure @SkyeYoung, I'll wait! 😊

Hi, I have tried to commit #1315 to fix the build issue in windows and it works fine in my windows 11 environment. (But with commands like `yarn prepare-data`, you still need to run them manually, and I haven't tried to get it to work automatically in windows for now.)

**Comment by BhavyaT-135** [comment]
Hi @SkyeYoung @yzeng25 , sorry for the delay. Got busy with some work at the university. I'll create some more thumbnail images for the blog posts. 

Meanwhile, should I create more thumbnails for this blog or should we go with one of these? 😊 

> Hi @SkyeYoung @yzeng25, I tried setting up the project on my local machine but am running into some issues. I have contributed to the project before and it was working fine back then. I don't understand what the problem is now. 😅
> 
> For the time being, I tried the preview of the two images on the official website using Google developer tools. Kindly take a look 😃
> 
> ![apisix-orange](https://user-images.githubusercontent.com/55613637/188588023-2e27cc82-6f16-4905-aabb-017c982a06f3.png)
> 
> ![apisix-green](https://user-images.githubusercontent.com/55613637/188588109-7879385d-2ddb-4338-b463-4b2eb0bf95d1.png)

**Comment by guitu168** [comment]
@BhavyaT-135  great! 👍 We go with one of these. Can the background color of the picture be white?

**Comment by BhavyaT-135** [comment]
Hi I created a few designs, because the white background one looked a little too simple. Could you please let me know your thoughts on them, @hf400159 ? 😊 

![apisix-eureka-3](https://user-images.githubusercontent.com/55613637/191015659-3ac441ba-f6b4-4945-a062-5f963c242d49.png)

![apisix-eureka-4](https://user-images.githubusercontent.com/55613637/191015738-21dd3758-8189-445b-a6fa-b65b521a5667.png)

![apisix-eureka-5](https://user-images.githubusercontent.com/55613637/191015767-d87faba5-21bc-45f4-9c4e-a401972b7865.png)

**Comment by guitu168** [comment]
The last picture is nice. Can you refer to these images too? The background color will add some other colors.  @BhavyaT-135 
![image](https://user-images.githubusercontent.com/97138894/191425476-3863bd92-201a-40cf-a067-7af9fef8cc36.png)
![image](https://user-images.githubusercontent.com/97138894/191425531-f997d743-18c7-4633-9834-87658b4e006e.png)
![image](https://user-images.githubusercontent.com/97138894/191425595-1f643434-e190-496f-a687-913c780b65f7.png)

**Comment by BhavyaT-135** [comment]
Hey @hf400159, thanks for the advice. I came up with this design. How does it look? 😄 

![apisix-eureka-6](https://user-images.githubusercontent.com/55613637/191530668-6716969f-b6b2-4f9b-9022-db87ffcab98d.png)

**Comment by guitu168** [comment]
@BhavyaT-135 Great！Welcome to submit PR.

**Comment by BhavyaT-135** [comment]
Hi @hf400159, thanks for the approval! 😊

Could you kindly guide me as to where to insert the image in the code-base as I couldn't seem to find the specific location. Also, do I need to upload the image on a specific link like the rest of the thumbnails? 

P.S., Sorry for bothering 😅

**Comment by yzeng25** [comment]
Hi @BhavyaT-135, these images are great. Just a small piece of advice, would you be able to focus on the content on the **first** page of the blog section first?

IMO the first page serves as an entry, it has to look good. The other pages are less important, which gives you plenty of time to solve them in batches, i.e. submit a pull request with 1~5 images at a time.

For image insertions and conversions, you can seek help from @SkyeYoung and @hf400159.

**Comment by guitu168** [comment]
> Hi @hf400159, thanks for the approval! 😊
> 
> Could you kindly guide me as to where to insert the image in the code-base as I couldn't seem to find the specific location. Also, do I need to upload the image on a specific link like the rest of the thumbnails?
> 
> P.S., Sorry for bothering 😅

@BhavyaT-135 You can upload pictures to the comments. I will convert these image's links.

**Comment by BhavyaT-135** [comment]
Sure @yzeng25 I'll start working on thumbnails of the first page!😊

> > Hi @hf400159, thanks for the approval! 😊
> > Could you kindly guide me as to where to insert the image in the code-base as I couldn't seem to find the specific location. Also, do I need to upload the image on a specific link like the rest of the thumbnails?
> > P.S., Sorry for bothering 😅
> 
> @BhavyaT-135 You can upload pictures to the comments. I will convert these image's links.
@hf400159 Great! I'll keep uploading them here!😊

**Comment by BhavyaT-135** [comment]
Hi @yzeng25 @hf400159, I've created the following thumbnails for the blog titled "**Why would you choose Apache APISIX instead of NGINX or Kong?**" 

Kindly take a look and let me know which one looks better. Open to changes😊 

![apache-apisix](https://user-images.githubusercontent.com/55613637/192514392-f77f8918-c419-4763-9b99-b69bd692b44f.png)

![apache-apisix1](https://user-images.githubusercontent.com/55613637/192514413-1561793b-b877-4c40-b884-b3bfd3c9c2e0.png)

**Comment by guitu168** [comment]
@BhavyaT-135 Great! I like the style of the first one. `>` means a switch from APISIX to Kong and NGINX? Can you change the `>` symbol?

**Comment by BhavyaT-135** [comment]
@hf400159 Actually the `>` symbol was just an arithmetic "greater than" sign. Sure, I'll update that ASAP. Should I change it to an arrow pointing towards apisix or a straight line like the previous designs. What do you suggest? 😃

**Comment by yzeng25** [comment]
I suggest using the second one to avoid ambiguities.

**Comment by guitu168** [comment]
> @hf400159 Actually the `>` symbol was just an arithmetic "greater than" sign. Sure, I'll update that ASAP. Should I change it to an arrow pointing towards apisix or a straight line like the previous designs. What do you suggest? 😃

How about using arrows? @BhavyaT-135 Image two looks informal. cc @SylviaBABY What do you think?

**Comment by BhavyaT-135** [comment]
@hf400159, I'm attaching the thumbnail with the arrow and the simple line in the middle one. Kindly take a look... 😊 

![apache-apisix2](https://user-images.githubusercontent.com/55613637/192825002-db92ee39-d9b8-4135-9828-52b52bb42bf2.png)

![apache-apisix3](https://user-images.githubusercontent.com/55613637/192825040-13c32e39-abcb-4fff-b322-0192d0cb8b3a.png)

**Comment by BhavyaT-135** [comment]
Hi @yzeng25 @hf400159, I've created the following design for the article titled "**Apache APISIX embraces the WASM ecosystem**". Kindly take a look and suggest necessary changes. 😊 

![apisix-WASM](https://user-images.githubusercontent.com/55613637/193191490-ba890dc8-7bea-4765-a0ef-8f47dc7d3800.png)

**Comment by BhavyaT-135** [comment]
Hey @hf400159 @yzeng25, I came up with another design for the article titled "**Apache APISIX loves Rust!**". Kindly take a look. 😊 

![apisix-rust](https://user-images.githubusercontent.com/55613637/193446526-ee89740f-f980-4432-b2f7-b619899cb822.png)

**Comment by guitu168** [comment]
https://static.apiseven.com/2022/10/08/634113b161cce.png
https://static.apiseven.com/2022/10/08/634113b21a4aa.png
https://static.apiseven.com/2022/10/08/634113b2e6978.png

@BhavyaT-135 This is the image link; you can change the blog cover image. great work! 👍

**Comment by BhavyaT-135** [comment]
Hey @hf400159, I've created a PR addressing the above three blogs' cover images. Kindly take a look! 😊

**Comment by BhavyaT-135** [comment]
Hi @yzeng25 @hf400159, hope you are in good health. I've created a design for the blog titled  "GCP, AWS, and Azure ARM-based server performance comparison". Kindly take a look and let me know your thoughts. 😊 

![apache-apisix](https://user-images.githubusercontent.com/55613637/196001692-d9b23177-021d-4918-b085-b8a467307c63.png)

**Comment by guitu168** [comment]
@BhavyaT-135 Excellent! you can add Oracle Cloud Infrastructure. 
please refer to: https://github.com/apache/apisix-website/pull/1360/files  
 Currently, only Chinese blogs are submitted.

**Comment by BhavyaT-135** [comment]
Sure @hf400159, does this look apt? 😊 

![apache-apisix2](https://user-images.githubusercontent.com/55613637/196195559-7bbf09b0-24ae-4f2b-abf9-1776ccb3190b.png)

**Comment by guitu168** [comment]
> Sure, @hf400159, does this look apt? 😊
> 

@BhavyaT-135 LGTM! The new Blog has not been published yet, so we need to wait. In the meantime, you can continue to create cover images for other blogs.

**Comment by BhavyaT-135** [comment]
Great! I'll get on with the other blogs! Thanks! 😊

**Comment by BhavyaT-135** [comment]
Hey @hf400159, I created the design for the article titled "Fault Injection Testing with API Gateway". Kindly check and let me know if there are any changes to be made. 😊 

![apache-apisix](https://user-images.githubusercontent.com/55613637/196392900-09118c9a-6203-4b72-aa5b-22ae7688dca7.png)

**Comment by SkyeYoung** [comment]
> Hey @hf400159, I created the design for the article titled "Fault Injection Testing with API Gateway". Kindly check and let me know if there are any changes to be made. 😊
> 
> ![apache-apisix](https://user-images.githubusercontent.com/55613637/196392900-09118c9a-6203-4b72-aa5b-22ae7688dca7.png)

LGTM, Great Design!

**Comment by guitu168** [comment]
@BhavyaT-135 LGTM! This is an image link!  you can create a PR. https://static.apiseven.com/2022/10/20/6350b3f63a715.png

**Comment by BhavyaT-135** [comment]
Created a PR addressing the same. 😊

**Comment by BhavyaT-135** [comment]
Hey @hf400159 @yzeng25 @SkyeYoung! Created a design for the article titled "API monetization using an API Management and a billing provider". Kindly take a look and let me know if this works. 😊 

![apisix-website-3](https://user-images.githubusercontent.com/55613637/197498179-50a33260-8196-4824-a2da-53a12414dcf8.png)

**Comment by BhavyaT-135** [comment]
Created another design for the article titled "Deploy and Run Apache APISIX on Scaleway Cloud". Please check! 😊 

![apisix-website-2](https://user-images.githubusercontent.com/55613637/197574659-92009f6b-b82a-4000-86e8-71e312c0e233.png)

**Comment by guitu168** [comment]
Great! I like your design very much. 👍 @BhavyaT-135 

https://static.apiseven.com/2022/10/25/6357addcaa23e.png
https://static.apiseven.com/2022/10/25/6357addd22a01.png

**Comment by BhavyaT-135** [comment]
Thanks @hf400159! Created a PR for the cover images! 😊

**Comment by BhavyaT-135** [comment]
Hey @hf400159, does this design look apt for the blog titled "Rewriting the Apache APISIX response-rewrite plugin in Rust"? Or is it a little too much? 😅 

![apisix-website](https://user-images.githubusercontent.com/55613637/198063095-ae0c3d58-c1e1-4efa-b940-1002ebadb5f6.png)

**Comment by guitu168** [comment]
@BhavyaT-135 Great design!

**Comment by BhavyaT-135** [comment]
Hi @hf400159! Created the design for the blog titled "How is the Azure ARM architecture server perform?". Kindly take a look and let me know if any changes are required. 😊 

![apisix-website](https://user-images.githubusercontent.com/55613637/198276961-ccb8da94-7dfa-4f27-9a0c-52169b90ead8.png)

**Comment by guitu168** [comment]
Hi @BhavyaT-135 ,  Great design! there are image links! For other cover images, Can you refer to the picture below to change a design style?
![image](https://user-images.githubusercontent.com/97138894/198500314-90cbb9e0-3204-47d7-a906-f2ae8c3f4911.png)
![image](https://user-images.githubusercontent.com/97138894/198500365-38502b73-e35c-4d09-8f6a-e2d0c1ea106d.png)

https://static.apiseven.com/2022/10/28/635b5378cdd1f.png
https://static.apiseven.com/2022/10/28/635b53794f32a.png
https://static.apiseven.com/2022/10/18/634e6963ea45f.png

**Comment by BhavyaT-135** [comment]
Thanks @hf400159! I'll keep the above-mentioned designs in mind for the next time. Created a PR for the image links given! 😊

**Comment by BhavyaT-135** [comment]
Hi @hf400159! 👋 Meanwhile, I created another thumbnail as per the new design for the blog titled "Building event-driven API services using CQRS, API Gateway and Serverless". Do let me know your thoughts about it! 😊 

![apisix-website](https://user-images.githubusercontent.com/55613637/198898160-a8c6aa82-f30e-4f6a-abba-c700e8e414e3.png)

**Comment by guitu168** [comment]
@BhavyaT-135 LGTM. cc @SylviaBABY What do you think?

**Comment by SylviaBABY** [comment]
> @BhavyaT-135 LGTM. cc @SylviaBABY What do you think?

That's great, so creative and graphic. Nice work！

**Comment by guitu168** [comment]
@BhavyaT-135 This is the image link! https://static.apiseven.com/2022/11/01/63612aab60c8d.png

**Comment by BhavyaT-135** [comment]
Hey @hf400159, I created two designs for the article titled "Backend-for-Frontend: the demo". I guess I went a little overboard with the first one... 😅 

![apisix-website](https://user-images.githubusercontent.com/55613637/199258262-8e13bda2-254a-4bb3-9920-4d31e68c80fa.png)

So, I created another design that was not too fancy. Kindly let me know which one suits better and suggest changes if required.😊 


![apisix-website-2](https://user-images.githubusercontent.com/55613637/199258514-87cec275-64fe-48fd-bd5a-05b56c1cea22.png)

**Comment by BhavyaT-135** [comment]
> @BhavyaT-135 This is the image link! https://static.apiseven.com/2022/11/01/63612aab60c8d.png

Created a PR for the same! 😊

**Comment by guitu168** [comment]
first image link: https://static.apiseven.com/2022/11/01/63613794464e6.png

second image link: https://static.apiseven.com/2022/11/01/63613794d16ea.png

@BhavyaT-135  I like the first image. But the second image is more APISIX style. cc @SylviaBABY

**Comment by SylviaBABY** [comment]
> first image link: https://static.apiseven.com/2022/11/01/63613794464e6.png
> 
> second image link: https://static.apiseven.com/2022/11/01/63613794d16ea.png
> 
> @BhavyaT-135 I like the first image. But the second image is more APISIX style. cc @SylviaBABY

I like the first one so much! It's so adorable. But as Fei said, the second will match the APISIX style, and I think maybe removing the words from the image will be great.

**Comment by BhavyaT-135** [comment]
Thanks @SylviaBABY @hf400159, I agree. Here's the thumbnail with the words removed. 😊 

![apisix-website-3](https://user-images.githubusercontent.com/55613637/199556582-8edcc937-815f-40f1-b0af-6eb006589121.png)

**Comment by guitu168** [comment]
@BhavyaT-135 This is the image link!  https://static.apiseven.com/2022/11/03/63634494405e7.png

**Comment by BhavyaT-135** [comment]
Hey @hf400159! I added the cover image for the "Backend-for-Frontend" blog in the same PR as before (#1387). 😊

**Comment by BhavyaT-135** [comment]
Hi @hf400159! I created a design for the blog titled "Why Is Apache APISIX the Best API Gateway?". Kindly take a look and let me know if it suits well. 😊 

![apisix-website](https://user-images.githubusercontent.com/55613637/200044601-5a4b9375-140e-4ff0-8196-65004068ef6b.png)

**Comment by guitu168** [comment]
LGTM, This is the image link. https://static.apiseven.com/2022/11/05/6365f2b8be5a7.png

**Comment by BhavyaT-135** [comment]
Hey @hf400159, I created another design for the blog titled "Why do you need Apache APISIX when you have NGINX and Kong?". Kindly check and suggest changes if required. 😊 

![apisix-website](https://user-images.githubusercontent.com/55613637/200140326-6320ca6e-125c-4a8e-902d-de111f097ed7.png)

**Comment by guitu168** [comment]
great! Can you add a caption directly in the middle of the picture?
![image](https://user-images.githubusercontent.com/97138894/200161132-3b674e83-343a-492b-baa9-e20fb8670a73.png)

**Comment by BhavyaT-135** [comment]
Hi @hf400159, does this look more apt?

![apisix-website-2](https://user-images.githubusercontent.com/55613637/200163160-20482c4a-fe28-4668-a120-9bc13b7e582f.png)

Or does the one without the logos look better? 😊 

![apisix-website-3](https://user-images.githubusercontent.com/55613637/200163341-8431cc7b-540a-4186-802a-2a40cfea05b1.png)

**Comment by guitu168** [comment]
I like the second one. What do you think? @SylviaBABY

**Comment by SylviaBABY** [comment]
> I like the second one. What do you think? @SylviaBABY

I agree

**Comment by guitu168** [comment]
hi, @BhavyaT-135 .This is the image link. https://static.apiseven.com/2022/11/07/636916ea65769.png

**Comment by BhavyaT-135** [comment]
Hey @hf400159! I added the thumbnail for this blog in the latest PR (#1395) as it hadn't been merged yet. 😊

**Comment by BhavyaT-135** [comment]
Hi @hf400159, I created a design for the blog titled "How is Google Cloud Tau T2A performing? ". Kindly take a look and suggest changes if required. 😊 

![apisix-website](https://user-images.githubusercontent.com/55613637/200504163-8ff40709-b58b-4b0a-8c18-8d13b34a91ad.png)

**Comment by guitu168** [comment]
hi, @BhavyaT-135 .This is the image link. https://static.apiseven.com/2022/11/11/636dc26637f24.png

**Comment by BhavyaT-135** [comment]
Hi @hf400159, I created a design for the blog titled "Getting Started with APISIX Test Cases". Kindly check and let me know if anything needs to be changed 😊 

![apisix-website](https://user-images.githubusercontent.com/55613637/201486590-8c5efdae-83c8-4cc1-8124-647534fef5cd.png)

**Comment by guitu168** [comment]
It's cool! @BhavyaT-135 This is the image link.  https://static.apiseven.com/2022/11/13/637053d67a0d4.png

**Comment by BhavyaT-135** [comment]
@hf400159 , I added the above mentioned cover image to the already open PR. 😊

**Comment by BhavyaT-135** [comment]
Hey @hf400159 👋, I created a design for the blog titled "APISIX Integrates ClickHouse to Improve Log Efficiency". Kindly take a look and let me know if any changes are required 😊 

![apisix-website](https://user-images.githubusercontent.com/55613637/202554073-115cee3a-184e-4b39-8927-9390e9ccf36a.png)

**Comment by BhavyaT-135** [comment]
Also, I wanted to ask what can we do about the Biweekly report blogs. This is the current thumbnail used for the latest biweekly report:

https://static.apiseven.com/apisix-webp/2022/weeklyreport/0809/biweekly-2022.7.16-2022.7.31-en.webp

I think it's a great design. So should we just use this for all biweekly reports and change the date ranges? Or should there be new designs for them? 😊

**Comment by guitu168** [comment]
@BhavyaT-135 This is the image link. https://static.apiseven.com/2022/11/18/63774d2d76267.png

The biweekly report is no longer updated, so keep this design for now.

**Comment by BhavyaT-135** [comment]
Great @hf400159, so should I go ahead and create a PR for each biweekly report with date changes in the same design? 😊

**Comment by BhavyaT-135** [comment]
Hi @hf400159, I created a design for the blog titled "How to monitor Apache APISIX with DataAnt". Kindly review 😊 

![apisix-website](https://user-images.githubusercontent.com/55613637/202843501-3025b8a8-ae23-4c07-8179-fb5330950e19.png)

**Comment by guitu168** [comment]
> Great @hf400159, so should I go ahead and create a PR for each biweekly report with date changes in the same design? 😊

No need, keep it as it is.

This is the image link. https://static.apiseven.com/2022/11/19/6378c1cf6c6c7.png

**Comment by BhavyaT-135** [comment]
Hey @hf400159, I created the following design for the blog titled "A Poor Man's API". Kindly review and suggest any changes if required 😊 

![apisix-website](https://user-images.githubusercontent.com/55613637/203798980-5e1cbd77-3868-4aec-9d38-9f61a2c802df.png)

**Comment by guitu168** [comment]
@BhavyaT-135 This is the image link.  https://static.apiseven.com/2022/11/28/63841cb4ad10a.png

**Comment by SkyeYoung** [comment]
@BhavyaT-135 Really did a long time from September to now, thank you so much. 👍🏻

**Comment by BhavyaT-135** [comment]
Thanks @SkyeYoung, it's my pleasure to contribute!😊

**Comment by BhavyaT-135** [comment]
Hi @hf400159, I created a thumbnail for the blog titled "Implementation of canary release solution based on Apache APISIX". Kindly review 😊 

![apisix-website](https://user-images.githubusercontent.com/55613637/205446492-893bb32c-b1ff-4b94-94ff-d805bec53f81.png)

**Comment by SylviaBABY** [comment]
> Hi @hf400159, I created a thumbnail for the blog titled "Implementation of canary release solution based on Apache APISIX". Kindly review 😊
> 
> ![apisix-website](https://user-images.githubusercontent.com/55613637/205446492-893bb32c-b1ff-4b94-94ff-d805bec53f81.png)

Thanks, @BhavyaT-135 , LGTM. And this is the image link: https://static.apiseven.com/2022/12/12/63969c5eb411d.png

**Comment by BhavyaT-135** [comment]
Thanks, @SylviaBABY, I've created a PR addressing the same😊

**Comment by Mumtaz12** [comment]
I would like to work on this issue

**Comment by Zerozero-123** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by SkyeYoung** [comment]
@Mumtaz12 Hi, sorry for being so late. You are welcome to contribute. @BhavyaT-135 has already done a lot of images, and you can browse through the images he's created for the community in the past to make new pictures.

**Comment by yzeng25** [comment]
All good now, Thanks everyone involved for helpful discussions and code contributions
